test/providers/google_cloud_dns_test.rb

```diff
@@ -16,8 +16,7 @@ def test_build_a_from_api
         'A',
         600,
         '10.11.12.13'
-      )
-    )
+      ))
```

Comment:
Please do the same in that file! (first parameter of the function in another line instead of merging the closing chars)

Reply:
This file has been updated :)